tachyon/zk/plonk/lookup/compress_expression.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020-2022 The Electric Coin Company
+// Copyright 2022 The Halo2 developers
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.halo2 and the LICENCE-APACHE.halo2
+// file.
+
+#ifndef TACHYON_ZK_PLONK_LOOKUP_COMPRESS_EXPRESSION_H_
+#define TACHYON_ZK_PLONK_LOOKUP_COMPRESS_EXPRESSION_H_
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "tachyon/zk/plonk/circuit/expressions/evaluator/simple_evaluator.h"
+
+namespace tachyon::zk {
+
+template <typename PCSTy>
+void EvaluateSingleExpression(
+    absl::Span<typename PCSTy::Field> values,
+    const Expression<typename PCSTy::Field>* expression,
+    const SimpleEvaluator<typename PCSTy::Evals>& evaluator_tpl) {
+  SimpleEvaluator<typename PCSTy::Evals> evaluator = evaluator_tpl;
+  for (size_t i = 0; i < values.size(); ++i) {
+    values[i] = evaluator.Evaluate(expression);
+  }
+}
```

Comment:
It would be great to `Expression` have it as a method.

Reply:
the function will be removed.